models/cnb_model.py

```diff
@@ -125,14 +125,14 @@ def _predictions_with_proba(
         array with the `IncidentType` prediction as the first element and the
         confidence as the second.
         """
-        top_indices: np.ndarray = proba.argsort()[:, -1 : -(num_predictions + 1) : -1]
+        top_indices: np.ndarray = proba.argsort()[:,
+                                                  -1:-(num_predictions + 1):-1]
```

Comment:
this styling is kind of gross... is this part of the linting stuff? @DhyeyLalseta

Reply:
yeah I think it is, I think it's because of[ this setting](https://github.com/Code-the-Change-YYC/YW-NLP-Report-Classifier/blob/7ef52018e5cd9fd95db01092709052efda8e2b3c/.style.yapf#L4) in the config. should I take it out and re-lint project?